src/models/user.rs

```diff
@@ -97,7 +99,8 @@ impl<'a> NewUser<'a> {
                     .optional()?;
 
                 if let Some(token) = token {
-                    crate::email::send_user_confirm_email(user_email, &user.gh_login, &token);
+                    // Swallows any error. Some users might insert an invalid email address here.
+                    let _ = emails.send_user_confirm(user_email, &user.gh_login, &token);
```

Comment:
from a layering perspective it feels a bit wrong to me to have email sending code in the database model struct, but I guess refactoring this is out of scope for this PR